app/controllers/app.go

```diff
@@ -1,11 +1,139 @@
 package controllers
 
-import "github.com/revel/revel"
+import (
+	"encoding/json"
```

Comment:
Please, let's use 4-space soft tabs, not literal tab characters for indentation.